utils/envs.go

```diff
@@ -40,18 +41,9 @@ func CheckEnvs(envs [][]string, reqEnvs []string) []string {
 	return missingEnvs
 }
 
-func GetDockerEnvFlags(envs [][]string) string {
-	var envFlag string
-	for _, pair := range envs {
-		envFlag = envFlag + " -e " + strings.Join(pair, "=")
-	}
-	return envFlag
-}
-
-func GetDockerEnvFlagss(envs [][]string) []string {
-	envFlags := []string{}
-	for _, pair := range envs {
-		envFlags = append(envFlags, " -e "+strings.Join(pair, "="))
-	}
+func GetDockerEnvFlags() string {
+	envFlags := " -e AWS_ACCESS_KEY_ID=" + viper.GetString("aws.access.key.id") +
```

Comment:
Currently in master, the kops commands built in kops.go no longer use this function to grab the environment variables. The binary runs because the values are exported in the makefile or, if running on a cluster, when the deployment is created. The variables that come through viper either need to be exported to make it work like it is in master right now (or maybe they are already being exported like you have it), or we need to add this function into the binary command so that the variables are there.